src/Illuminate/Validation/Validator.php

```diff
@@ -747,7 +747,7 @@ protected function validateDifferent($attribute, $value, $parameters)
 
         $other = Arr::get($this->data, $parameters[0]);
 
-        return isset($other) && $value != $other;
+        return isset($other) && $value !== $other;
```

Comment:
I guess this probably makes sense. You wouldn't really compare a string vs
an integer I don't think.

On Sunday, June 28, 2015, Nekrasov Ilya <notifications@github.com> wrote:

> In src/Illuminate/Validation/Validator.php
> <https://github.com/laravel/framework/pull/9439#discussion_r33427452>:
>
> > @@ -747,7 +747,7 @@ protected function validateDifferent($attribute, $value, $parameters)
> >
> >          $other = Arr::get($this->data, $parameters[0]);
> >
> > -        return isset($other) && $value != $other;
> > +        return isset($other) && $value !== $other;
>
> @taylorotwell <https://github.com/taylorotwell> are you strongly against
> or should I resend this to 5.2?
>
> To my mind the fact that you can confirm your "1000" password with "1e3"
> is pretty meh.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/framework/pull/9439/files#r33427452>.
>